src/radical/pilot/agent/lm/orte.py

```diff
@@ -180,9 +184,12 @@ def lrms_shutdown_hook(cls, name, cfg, lrms, lm_info, logger):
                 orterun = ru.which('orterun')
                 if not orterun:
                     raise Exception("Couldn't find orterun")
-                subprocess.Popen([orterun, "--hnp", lm_info['dvm_uri'], "--terminate"])
+                ru.sh_callout('%s --hnp %s --terminate' 
+                             % (orterun, lm_info['dvm_uri']))
+                profiler.prof(event='orte_dvm_stop', uid=cfg['pilot_id'])
             except Exception as e:
-                logger.exception('dmv termination failed')
+                logger.exception('dvm termination failed')
+                profiler.prof(event='orte_dvm_fail', uid=cfg['pilot_id'])
```

Comment:
Do you think it is correct to have the same event name between a `ORTE DVM` failure and a shutdown failure?

Reply:
Actually yes - since the DVM is running asynchronously, we can't really distinguish a shutdown failure from a runtime failure.  In fact, the most common case where we see the shutdown failure is when the shutdown complains that the DVM is already gone, which is then indeed a runtime failure.  So at the moment this is intentional for the profiler, but the logs will distinguish it.  I added a comment in 64cf2da5 to clarify this.